docs/naps/2-conda-based-packaging.md

```diff
@@ -0,0 +1,527 @@
+
+
+### Milestone 1: Distributing napari and plugins on conda-forge
+
+This work can be divided in two different tasks: adjusting the conda-forge feedstock for napari,
+and then migrating all the plugins over to conda-forge.
+
+* [Adjust outputs for multiple qt backends and menuinst=2](https://github.com/conda-forge/napari-feedstock/pull/32)
+* [Plugin migration search query](https://github.com/conda-forge/staged-recipes/pulls?q=is%3Apr+author%3Agoanpeca+created%3A2022-01-01..2022-05-01)
+  (~200 PRs)
+
+At the time of writing, the plugin migration to conda-forge can be considered 90% done, but a long
+tail of non-trivial packages need to be worked on. This is caused by vendored binaries,
+non-compliant licensing schemes or bad packaging practices. Looking forward, a set of guidelines
+in the plugin development documentation should be added.
+
+### Milestone 2: Building conda-based installers for napari
+
+Implementing the `constructor` workflow on napari was mainly done in a single, long-lived PR, that
+has seen a couple of minor updates in the recent releases:
+
+* Main PR: https://github.com/napari/napari/pull/3378, superseded by
+  https://github.com/napari/napari/pull/3555
+* Other PRs:
+    * https://github.com/napari/napari/pull/3462
+    * https://github.com/napari/napari/pull/4185
+    * https://github.com/napari/napari/pull/4210
+    * https://github.com/napari/napari/pull/4221
+    * https://github.com/napari/napari/pull/4307
+    * https://github.com/napari/napari/pull/4309
+    * https://github.com/napari/napari/pull/4387
+    * https://github.com/napari/napari/pull/4444
+    * https://github.com/napari/napari/pull/4447
+
+
+Adding the missing pieces to `constructor` involves changes in four different projects:
+
+* `conda/constructor` itself, the tool that builds the installer
+* `conda/menuinst`, responsible of creating the shortcuts
+* `conda/conda`, which relies on `menuinst` to delegate the shortcut creation, but this code path
+  was only enabled on Windows so far
+* `conda-forge/conda-standalone-feedstock`, which freezes a `conda` install along with their
+  dependencies and adds a thin layer on top to communicate with the `constructor` installer during
+  the installation process.
+
+The full list of PRs is available on
+[this issue comment](https://github.com/napari/napari/issues/4248#issuecomment-1066574137).
+A complete list of features can be found in the packaging documentation [^napari-packaging-docs].
+
+### Milestone 3: Adding support for conda packages in the plugin manager
+
+Support for conda/mamba handling of plugin installations was implemented in a base
+PR and then extended with different PRs:
+
+* Initial PR: https://github.com/napari/napari/pull/2943
+* Other PRs that improved and extended the funcionality:
+    * https://github.com/napari/napari/pull/4074
+    * https://github.com/napari/napari/pull/3369
+    * https://github.com/napari/napari/pull/3288
```

Comment:
I know it's somewhat redundant and annoying, but could you add the titles for these? It will make it easier to just read the document in one go. e.g. `[#3288](https://github.com/napari/napari/pull/3288): Fix updating of plugins`. (Though in retrospect maybe that should have been "fix updating of plugins when conda-forge version lags PyPI version", but we'll get there one day :joy:)